reflect-kotlin/src/test/java/dagger/DaggerTest.kt

```diff
@@ -0,0 +1,14 @@
+package dagger
+
+import org.junit.Assert.assertNotNull
+import org.junit.Test
+
+class DaggerTest {
+  @Test fun create() {
+    assertNotNull(NoBuilderComponent::class.create())
+  }
+
+  @Test fun builder() {
+    assertNotNull(BuilderComponent.Builder::class.builder())
```

Comment:
since there are no dependencies `BuilderComponent::class.create()` should also work?